README.md

```diff
@@ -4,7 +4,7 @@ Mojo::WebSocketProxy::Backend::ConsumerGroups - Class for communication with bac
 
 # VERSION
 
-version 0.02
+version 0.04
```

Comment:
Why are we jumping to version to 0.04 instead of 0.03?

Reply:
there is already 0.03 which the dev forget to update the version on binary repo, so when we made release the version upgraded to 0.04